common/errors.go

```diff
@@ -0,0 +1,5 @@
+package common
```

Comment:
I wonder why is this needed? Is there a problem to use the sql not found error due to circular dependency?

Reply:
I didn't want to use `sql.ErrNotFound` for the AtxService because it's not necessarily coming from a database. A DB-based implementation is one of many possible, see the #6355 for another one, for which SQL error made no sense

Reply:
And why not have an error which is defined in the activation package then? It can be as granular as `atx not found` then, no?

Reply:
Refactored:
- removed common/errors.go,
- changed to return the existing `activation.ErrNotFound`,
- added documentation about the returned error to the `AtxService` interface definition.